docs/installation/importing-css-assets-and-javascript.md

```diff
@@ -0,0 +1,162 @@
+# Import CSS, assets and JavaScript
+
+## CSS
+
+### Import all the CSS
+
+To import all the CSS styles from GOV.UK Frontend, add the following to your Sass file:
+
+```Scss
+@import "node_modules/govuk-frontend/govuk/all";
+```
+
+### Import an individual component's CSS
+
+To import the [button](https://design-system.service.gov.uk/components/button/) component for example, add the following to your Sass file:
+
+```SCSS
+@import "node_modules/govuk-frontend/govuk/components/button/button";
+```
+
+### Simplify Sass import paths
+
+If you want to make Sass import paths shorter, add `node_modules/govuk-frontend` to either your:
+
+- [Sass load paths](https://sass-lang.com/documentation/at-rules/import#finding-the-file)
+- [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) if you use Ruby in your project
+
+You can then import only the [button](https://design-system.service.gov.uk/components/button/) component for example, by adding the following to your Sass file:
+
+```SCSS
+@import "govuk/components/button/button";
+```
+
+### Override with your own CSS
+
+If you want to override GOV.UK Frontend's styles with your own styles, `@import` GOV.UK Frontend's styles before your own Sass rules.
+
+### Using GOV.UK Frontend with our old frameworks
+
+If your project uses GOV.UK Frontend toolkit, GOV.UK Template or GOV.UK Elements, you can [configure GOV.UK Frontend to work with them](/docs/installation/compatibility.md).
+
+## Font and image assets
+
+To use the font and image assets from GOV.UK Frontend, you can either:
+
+- route requests to the GOV.UK Frontend package - recommended
+- copy the font and images into your application
+
+### Route requests - recommended
+
+Route requests from `<YOUR-SITE-URL>/assets` to `/node_modules/govuk-frontend/govuk/assets`.
```

Comment:
I think this section could still do with improving…

- I'm not sure that people understand what we mean by 'route requests' – even with Express, which is the example we're providing, that's not terminology they use on https://expressjs.com/en/starter/static-files.html
- If the user is using webpacker then the best solution doesn't look like either of these (as I understand it, they'd want to use `require.context` and override `$govuk-font-url-function` and `$govuk-image-url-function`)

This could be something to iterate on in the future though.

Reply:
Good point! Following our chat I've improved the content to address your first point. Let's iterate on the second bullet in future if that's ok?

Reply:
I love everything about this change.